FreeRTOS/FreeRTOS-Plus-TCP
# Follow Security Guide to update release.yml

Follow Security Guide to update release.yml

Description
-----------

Follow [Security Guide](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#understanding-the-risk-of-script-injections) to update release.yml.

Test Steps
-----------

Execute release flow in local branch, see result by https://github.com/ActoryOu/FreeRTOS-Plus-TCP/actions/runs/11609414461.

Checklist:
----------


- [NA] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
